packages/sdk/src/types/runtime/userGroup.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Context } from '.';
+
+/**
+ * A set of user group IDs. User groups are used to selectively enable several different
+ * properties of actors based on the memberships of the viewing user. See [[User.groups]],
+ * [[Appearance.enabled]].
+ */
+export default class UserGroup extends Set<string> {
+    // tslint:disable:no-bitwise
+
+    private context: Context;
+
+    constructor(initialContents: Iterable<string> = null, context: Context = null) {
+        super(initialContents);
+        if (context) {
+            this.setContext(context);
+        }
+    }
+
+    public setContext(c: Context) {
```

Comment:
>setContext [](start = 11, length = 10)

This needs to be hidden. But, I'm not sure it's even needed. Can we just always pass context to the constructor? When is it desired to not have a context reference here?